frontend/src/modules/activity/components/activity-form-drawer.vue

```diff
@@ -385,3 +387,9 @@ export default {
   name: 'AppActivityFormDrawer',
 };
 </script>
+
+<style scoped>
+.isError {
+  margin-bottom: 40px;
+}
```

Comment:
Is this needed? Since you are already applying the classes on top?
Also, instead of adding the above `:class="[$v.activityType.$errors.length ? 'isError' : 'mb-4']"` to each form item, would it be better to add to the <style scoped> the following?
```
.el-form-item.is-error {
   @apply mb-8;
}
```